### DISCUSSION TITLE
KEDA Chart v2.9.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#344] [Announcements]
Chart: v2.9.0 | App: v2.9.0

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm install keda kedacore/keda
```

### Features

- Support for KEDA v2.9.0 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.9.0))
- Introduce support to disable keep-alive (https://github.com/kedacore/charts/pull/335)
- Introduce support for AWS IRSA  (#319)
- Introduce support for configuring metrics API server port (#308)
- Introduce Kubernetes Service for KEDA operator (#337)
- Introduce the capability to restrict access from Kubernetes secrets (https://github.com/kedacore/charts/pull/320)
- Introduce the capability to add additional annotations (https://github.com/kedacore/charts/pull/334)
- Add Kubernetes labels for Azure AD Workload Identity (#328)

### Fixes / Changes

- Kubernetes v1.23 or above is required
- Use `policy/v1` for Pod Disruption Budget (https://github.com/kedacore/charts/pull/331)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @christidis made their first contribution in https://github.com/kedacore/charts/pull/286
* @doctorpangloss made their first contribution in https://github.com/kedacore/charts/pull/308
* @StevenJDH made their first contribution in https://github.com/kedacore/charts/pull/319
* @schelv made their first contribution in https://github.com/kedacore/charts/pull/322
* @blottn made their first contribution in https://github.com/kedacore/charts/pull/332
* @kevinteng525 made their first contribution in https://github.com/kedacore/charts/pull/320
* @penghuazhou made their first contribution in https://github.com/kedacore/charts/pull/335

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.8.2...v2.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.9.0'>KEDA Chart v2.9.0</a>.</em>

**Comment by v-shenoy** [comment]
Commenting here because there does not seem to be a separate discussion for release 2.9.3. 
@JorTurFer @zroubalik @tomkerkhove The v2.9.3 chart release refers to app version v2.9.2. What is that chart version number for app version 2.9.3?

**Reply by JorTurFer** [reply]
I'm not sure what has happened but I can't see the chart 2.9.4 (which I think is the version for app 2.9.3)

**Reply by JorTurFer** [reply]
I can see the chart merged https://github.com/kedacore/charts/pull/386
Maybe, did I forget to create the release?

**Reply by tomkerkhove** [reply]
Let's create one!

**Reply by JorTurFer** [reply]
I mean, the GH release

**Reply by JorTurFer** [reply]
Can you do it? I am in a meeting the whole day

**Reply by v-shenoy** [reply]
Are there any changes in the charts, by the way? (I expect the only change will be the version number)

**Reply by JorTurFer** [reply]
The version is already released, I can see https://github.com/kedacore/charts/pull/386.
I think that only the GH release is left. Can't you pull chart v2.9.4 using helm even if the GH release is missing?

**Reply by v-shenoy** [reply]
I was actually trying to checkout the GH tag, but I couldn't find one. I can pull the chart though, thanks!

**Reply by JorTurFer** [reply]
Nice!
In any case, could you create the GH release @tomkerkhove ?

**Reply by tomkerkhove** [reply]
Available on https://github.com/kedacore/charts/releases/tag/v2.9.4
